src/board.py

```diff
@@ -62,13 +62,81 @@ def add_marble_to_board(board, user_value):
     # Detect if user_value is a valid coords for our board.
     position = get_position_if_valid(board, user_value)
 
-    #If no, raise exception
+    # If no, raise exception
     if position == None:
         raise ValueError("Position given is not correct")
 
-    #Else, return board with new added value
+    # Else, return board with new added value
     board = deepcopy(board)
     board[position] = 1
     return board
 
+"""
+get_slice_boundaries_from_rotation_key get an integer and return tuple of slices for a 2d range.
+
+rotation_key: int 1 to 8
+
+1 & 2 rotate quarter 1.
+2 & 3 rotate quarter 2...
+Each quarter has 2 possibilites of rotation (counter clockwise & clockwise).
+"""
+def get_slice_boundaries_from_rotation_key(rotation_key):
```

Comment:
this function is hard to understand at first, we have to read the function which uses it.